docs/api.rst

```diff
@@ -0,0 +1,36 @@
+Developer Interface for FoxPuppet
+*********************************
+
+This part of the documentation describes the interfaces for using FoxPuppet.
+
+.. py:module:: foxpuppet.foxpuppet
+
+.. autoclass:: FoxPuppet
+   :inherited-members:
+
+
+Window Manager
+==============================
+
+.. py:module:: foxpuppet.windows
+
+.. autoclass:: WindowManager
+  :inherited-members:
+
+
+Base Window Model
+=================
+
+.. py:module:: foxpuppet.windows.base
+
+.. autoclass:: BaseWindow
+  :inherited-members:
+
+
+Browser Model
+=============
+
+.. py:module:: foxpuppet.windows.browser
+
+.. autoclass:: BrowserWindow
+  :inherited-members:
```

Comment:
I don't think so, users of a Python client will not think of this as a directory, but as a package/module.